docs/components/best-practices/architecture/deciding-about-your-stack-c7.md

```diff
@@ -76,7 +76,7 @@ There are several _advantages using the greenfield stack_:
 
 ### Considering Camunda Platform 8 instead
```

Comment:
```suggestion
### Considering Camunda 8 instead
```

Reply:
This PR is solely for Guides content and links referencing Guides content -- Best Practices will be reviewed in a separate PR 👍